diesel/src/pg/expression/expression_methods.rs

```diff
@@ -3726,4 +3726,19 @@ pub(in crate::pg) mod private {
 
     impl<T> RecordOrNullableRecord for Record<T> {}
     impl<T> RecordOrNullableRecord for Nullable<Record<T>> {}
+
+    pub trait CombinedOneNullableValue<O, Out>: SingleValue {
```

Comment:
Can we name this `CombinedAllNullableValue` or something like that instead to transport that this only evaluates to `Nullable` if all inputs are `Nullable`?

Reply:
Hi @weiznich , sure I will change the name accordingly. And yes they allarenullable only returns nullable value when both values are null, completing the logic we need for this trait.